src/PowerJoinClause.php

```diff
@@ -150,21 +138,19 @@ protected function useAliasInWhereBasicType(array $where): array
     {
         $table = $this->tableName;
 
-        if (Str::startsWith($where['column'] . '.', $this->alias . '.')) {
+        if (Str::startsWith($where['column'].'.', $this->alias.'.')) {
             return $where;
         }
 
         if (Str::contains($where['column'], $table)) {
             // if joining the same table, only replace the correct table.key pair
-            $where['column'] = str_replace($table . '.', $this->alias . '.', $where['column']);
```

Comment:
These two statements, where exactly the same

Reply:
True, but by just removing the `else` condition, we are likely breaking functionality. We probably need to remove the whole `if (Str::contains($where['column'], $table))` and keep the `$where['column'] = str_replace($table.'.', $this->alias.'.', $where['column']);` line

Reply:
My mistake. I removed the 'if' statement at all. And of course I can revert to its initial state too.